test/functional/page_objects/common_page.ts

```diff
@@ -257,7 +257,7 @@ export function CommonPageProvider({ getService, getPageObjects }: FtrProviderCo
           return currentUrl;
         });
 
-        await retry.try(async () => {
+        await retry.tryForTime(defaultFindTimeout, async () => {
```

Comment:
@LeeDr I think we should shorten timeout for url validation from 120 sec = `try: Joi.number().default(120000)` to 10 sec =`find: Joi.number().default(10000)`:  the logic here is that we wait for url to stop changing.
My guess is that it shouldn't take too long for re-direction/url change, and if it is happening we better fail fast.

Reply:
>Not sure how, but ciGroups ran 1-2 min faster vs my other PR with no navigation changes.

guess it is related

Reply:
When reducing any timeouts, we just have to be careful about running tests on Cloud.  I think Kibana can appear to stall when an index snapshot is happening.

Reply:
Yes, this could be the case. Let's see how it goes, but the global navigation timeout remains the same.

Reply:
I mean, I don't expect it to fail on cloud and if it will, then we should check performance of cloud instances.